src/GafferScene/Instancer.cpp

```diff
@@ -543,14 +637,12 @@ class Instancer::EngineData : public Data
 				return -1;
 			}
 
-			if( m_hasDuplicates )
+			if( m_indicesInactive.size() )
 			{
 				// If there are duplicates in the id list, then some point indices will be omitted - we
-				// need to check each point index to see if it got assigned an id correctly
+				// need to check each point index to see if it got assigned an id correctly // TODO
```

Comment:
What's the TODO?

Reply:
Oh, just an out of date comment. Fixed now.